client/angular-cli.json

```diff
@@ -18,9 +18,12 @@
       "prefix": "op",
       "mobile": false,
       "styles": [
+        "../node_modules/intro.js/introjs.css",
```

Comment:
Did you try to check for a introjs typescript version?
Try `npm install --save-dev @types/intro.js` (or `npm install --save-dev @types/introjs`)